### PR TITLE
Startisback.Startisback version 2.9.15

### PR DESCRIPTION
**Pull request by ghost**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/26910)

### CODE REVIEW

**Comment by BajajAditya123** [comment]
_`$ s update beluga.zip`_

**Comment by BajajAditya123** [comment]
pleaseplease

**Comment by BajajAditya123** [comment]
please please please please please please please please please

**Comment by BajajAditya123** [comment]
I HAVE DONE IT

**Comment by BajajAditya123** [comment]
@msftbot I have put Silent or SilentWithProgress and validate it too. So what would be the error

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
I'm stuck here:
image

File: manifests/s/Startisback/Startisback/2.9.15/Startisback.Startisback.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Startisback.Startisback
```
Comment (ItzLevvie) [inline manifests/s/Startisback/Startisback/2.9.15/Startisback.Startisback.installer.yaml:4]:
```suggestion
PackageIdentifier: StartIsBack.StartIsBack
```

File: manifests/s/Startisback/Startisback/2.9.15/Startisback.Startisback.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Startisback.Startisback
```
Comment (ItzLevvie) [inline manifests/s/Startisback/Startisback/2.9.15/Startisback.Startisback.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: StartIsBack.StartIsBack
```

File: manifests/s/Startisback/Startisback/2.9.15/Startisback.Startisback.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Startisback.Startisback
```
Comment (ItzLevvie) [inline manifests/s/Startisback/Startisback/2.9.15/Startisback.Startisback.yaml:4]:
```suggestion
PackageIdentifier: StartIsBack.StartIsBack
```

File: manifests/s/Startisback/Startisback/2.9.15/Startisback.Startisback.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Startisback.Startisback
+PackageVersion: 2.9.15
+PackageLocale: en-US
+Publisher: Startisback
+PackageName: Startisback++
```
Comment (ItzLevvie) [inline manifests/s/Startisback/Startisback/2.9.15/Startisback.Startisback.locale.en-US.yaml:8]:
```suggestion
PackageName: StartIsBack++
```

File: manifests/s/Startisback/Startisback/2.9.15/Startisback.Startisback.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Startisback.Startisback
+PackageVersion: 2.9.15
+PackageLocale: en-US
+Publisher: Startisback
```
Comment (ItzLevvie) [inline manifests/s/Startisback/Startisback/2.9.15/Startisback.Startisback.locale.en-US.yaml:7]:
```suggestion
Publisher: startisback.com
```

File: manifests/s/Startisback/Startisback/2.9.15/Startisback.Startisback.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Startisback.Startisback
+PackageVersion: 2.9.15
+PackageLocale: en-US
+Publisher: Startisback
+PackageName: Startisback++
+License: Proprietary
+ShortDescription: It can change start meny to classic start menu and taskbar too
```
Comment (ItzLevvie) [inline manifests/s/Startisback/Startisback/2.9.15/Startisback.Startisback.locale.en-US.yaml:10]:
```suggestion
ShortDescription: StartIsBack fully integrates with Windows and provides original Windows 7 start menu and taskbar experience, enhanced with many new features as well.
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/s/Startisback/Startisback/2.9.15/Startisback.Startisback.installer.yaml [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+PackageVersion: 2.9.15
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: exe
+  Scope: machine
```
Comment (KaranKad) [inline manifests/s/Startisback/Startisback/2.9.15/Startisback.Startisback.installer.yaml:14]:
```suggestion
  Scope: user
```

File: manifests/s/Startisback/Startisback/2.9.15/Startisback.Startisback.installer.yaml [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+  InstallerUrl: https://s3.amazonaws.com/startisback/StartIsBackPlusPlus_setup.exe
+  InstallerSha256: 0185FD9C1AA559EEBE84119920D7A967E702BA3B57B8E0D837F4F7B79108D719
+  InstallerSwitches:
+    Silent: /s
+    SilentWithProgress: /s
```
Comment (KaranKad) [inline manifests/s/Startisback/Startisback/2.9.15/Startisback.Startisback.installer.yaml:19]:
```suggestion
    Silent: /silent
    SilentWithProgress: /silent
```

**Review by KaranKad — CHANGES_REQUESTED** [review]

File: manifests/s/Startisback/Startisback/2.9.15/Startisback.Startisback.locale.en-US.yaml [hunk 9]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Startisback.Startisback
+PackageVersion: 2.9.15
+PackageLocale: en-US
+Publisher: Startisback
+PackageName: Startisback++
+License: Proprietary
```
Comment (KaranKad) [inline manifests/s/Startisback/Startisback/2.9.15/Startisback.Startisback.locale.en-US.yaml:9]:
```suggestion
License: Proprietary
LicenseUrl: https://www.startisback.com/license.txt
PackageUrl: https://www.startisback.com/
Moniker: startisback
Tags:
- start-menu
- customization
```

**Review by KaranKad — CHANGES_REQUESTED** [review]

**Pull request closed without merging**